### PR TITLE
Feature/magento review rating labels

### PR DESCRIPTION
**Pull request by action-simon**

### CODE REVIEW

**Comment by paales** [comment]
Thanks for the pull request Simon! Love seeing a contribution 😄 

Could you explain a little bit what you are trying to achieve?
Do you have examples of other sites that have a similar feature?
Do you think this emulates Magento' rating functionality closely enough?
Are the labels general enough?
Is this something that every graphcommerce user, using reviews wants?
If not, a substantial portion of users?
Should this functionality be configurable 'productRatingLabels'?

Technical:
It doesn't seem to be working as expected the 'quality' label is the same for all rating fields if you hover over any of the rating fields. Maybe move the label logic to inside StarRatingField?

Scherm­afbeelding 2023-07-05 om 09 49 23

File: packages/magento-review/components/CreateProductReviewForm/CreateProductReviewForm.tsx [hunk 1]
```diff
@@ -36,8 +36,22 @@ const parts = [
...
+    1: <Trans id='Useless' />,
+    2: <Trans id='Poor' />,
+    3: <Trans id='Ok' />,
+    4: <Trans id='Good' />,
+    5: <Trans id='Excellent' />,
+  }
+
+  function getLabelText(labelValue: number) {
+    return `${labelValue} Star${labelValue !== 1 ? 's' : ''}, ${labels[labelValue]}`
+  }
```
[the remark this answers is not in the document]
Comment (paales) [reply packages/magento-review/components/CreateProductReviewForm/CreateProductReviewForm.tsx:49]: Translations don’t work outside the component scope, the context isn’t set up there

**Comment by paales** [comment]
@action-simon What do you want to do with this PR?

**Comment by action-simon** [comment]
Greetings, if you wish, you may proceed with closing it for the time being. I will work on it later, and then I will open a new pr!

**Comment by paales** [comment]
Awesome :)

**Pull request closed without merging**